noir-projects/aztec-nr/value-note/src/value_note.nr

```diff
@@ -57,6 +57,20 @@ impl ValueNote {
     }
 }
 
+impl Deserialize<7> for ValueNote {
+    /// The following method needed to be implemented because the note is returned from a contract function in noir (e.g. in a TXe test)
```

Comment:
```suggestion
    /// The following method needed to be implemented because the note is returned from a contract function in noir (e.g. in a TXE test)
```